app/views/pages/_card.html.slim

```diff
@@ -2,10 +2,13 @@
   .hf-card
     a href=card.path
       .hf-card-container
-        = card_thumbnail(card)
-
+        = image_tag(card.thumbnail.url, class: 'full-width-image', alt: card.thumbnail.description, title: card.thumbnail.title)
```

Comment:
It looks like you are unintentionally overwriting some recent changes here. This should be how it was originally for this line with `card_thumbnail(card)`